src/platform/silabs/wifi/icd/WifiSleepManager.cpp

```diff
@@ -32,14 +32,14 @@ namespace {
  */
 CHIP_ERROR ConfigureLIBasedSleep()
 {
+    VerifyOrReturnError(ConfigureBroadcastFilter(true) == SL_STATUS_OK, CHIP_ERROR_INTERNAL,
+                        ChipLogError(DeviceLayer, "Failed to configure broadcasts filter."));
+
```

Comment:
Why the switch of order?

Reply:
It is because the API will fail if invoked after going to sleep and it is aligned with what is provided in the sample application

Reply:
I don't have an issue with switching them but Selective Listening works with the current implementation so in seems to work in practice; at least for the 917 SoC. 

Is the change specific due to limitations of the 9116?

Reply:
I would think it is better and cleaner to invoke all the operations before going to sleep.

I guess 917 SoC does not show this problem, technically the API should be configurable even after sleep has been invoked but in practice NCP combinations are having the issue. This included 917 NCP.

Reply:
Can we open follow up issues with the wiseconnect team to review this? The documentation doesn't specify and there should be no API difference between SoC and NCP.

Can you add a comment in the comment to specify that the sleep command needs to be last thing called?